style.css

```diff
@@ -0,0 +1,103 @@
+header {
```

Comment:
loveee the global stylesheet to keep a universal design system across the entire site!